libs/application/template-api-modules/src/lib/modules/shared/shared.service.ts

```diff
@@ -205,4 +213,39 @@ export class SharedTemplateApiService {
         return data?.applicationPaymentStatus
       })
   }
+
+  async addAttachment(
+    applicationId: string,
+    fileName: string,
+    buffer: Buffer,
+    uploadParameters?: {
+      ContentType?: string
+      ContentDisposition?: string
+      ContentEncoding?: string
+    },
+  ): Promise<string> {
+    const uploadBucket = getConfigValue(
+      this.configService,
+      'attachmentBucket',
+    ) as string
+
+    const uploadParams = {
+      Bucket: uploadBucket,
+      Key: fileName,
```

Comment:
This might override the same "Kvörtun.pdf" file for all users in the root.

The S3 key structure on the application system is something like this (please double check):

`{applicationId}/{uuid}-{fileName}`?

Reply:
Ahh of course missed that when going between buckets.